src/PKSim.Infrastructure/Services/PresenterLogger.cs

```diff
@@ -45,12 +40,12 @@ private void writeMessage(LogLevel logLevel, string name, int eventIdId, string
 
       public bool IsEnabled(LogLevel logLevel)
       {
-         return logLevel >= _logLevel;
+         return true;
```

Comment:
That looks wrong here. This is a way to filter what is being logged to the view. Unless this was never used..

Reply:
@msevestre @georgeDaskalakis We do not need to keep track of this anymore. By setting the minimum log level with builder.SetMinimumLevel when configuring the LoggingBuilder we ensure the log method will not be called at all. We could still use the logLevel for filtering when in combination to something more but that is not the case now so we remove it for now and the configuration step will also be smoother.